pkg/virt-handler/vm.go

```diff
@@ -1550,10 +1551,6 @@ func (c *VirtualMachineController) calculateLiveMigrationCondition(vmi *v1.Virtu
 		return newNonMigratableCondition(err.Error(), v1.VirtualMachineInstanceReasonCPUModeNotMigratable), isBlockMigration
 	}
 
-	if util.IsVMIVirtiofsEnabled(vmi) {
```

Comment:
This is tricky as older launcher's can be managed by newer virt-handler. Making them appear as migratable. @germag Does libvirt throws meaningful error in this case?